nav2_controller/src/controller_server.cpp

```diff
@@ -225,6 +232,7 @@ ControllerServer::on_configure(const rclcpp_lifecycle::State & state)
 
   odom_sub_ = std::make_unique<nav2_util::OdomSmoother>(node, odom_duration, odom_topic);
   vel_publisher_ = std::make_unique<nav2_util::TwistPublisher>(node, "cmd_vel");
+  global_path_pub_ = node->create_publisher<nav_msgs::msg::Path>("received_global_plan");
```

Comment:
Why this? We in other contexts publish out the path after its been pruned and converted into the odom frame for the local_path publisher so that we have a sense of what the controller is working with. Why directly publish out the global plan?